db/createItem.go

```diff
@@ -0,0 +1,44 @@
+package db
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/mount-joy/thelist-lambda/data"
+)
+
+func (d *dynamoDB) CreateItem(listID string, name string) (*data.Item, error) {
+	itemID := d.generateID()
+
+	item := &data.Item{
+		ListID: listID,
+		ID:     itemID,
+		Name:   name,
+	}
+	itemToInsert, err := dynamodbattribute.MarshalMap(item)
+	if err != nil {
+		return nil, err
+	}
+
+	input := &dynamodb.PutItemInput{
+		Item:                itemToInsert,
+		TableName:           aws.String(d.conf.TableNames.Items),
```

Comment:
In one the other PRs you checked that `d.conf.TableNames.Items` response had len not 0.
Up to you if you do that here too